examples/1d/plot_filters.py

```diff
@@ -21,144 +23,112 @@
-  # `phi_f[1]` corresponds to the filter at resolution `N/2`, and so on.
-  #
-  # While `phi_f` only contains a single filter (at different resolutions),
-  # the `psi1_f` and `psi2_f` outputs are lists of filters, one for each wavelet
-  # bandpass filter in the filter bank.
-  
-  ###############################################################################
-  # Plot the filters
-  # ================
-  # We are now ready to plot the filters. We first display the lowpass filter
-  # (at full resolution) in red. We then plot each of the bandpass filters in
-  # blue. Since we don't care about the negative frequencies, we limit the
-  # plot to the frequency interval :math:`[0, 0.5]`. Finally, we add some
-  # explanatory labels and title.
-  
-  plt.figure()
-  plt.plot(np.arange(N)/N, phi_f[0], 'r')
-  
-  for psi_f in psi1_f:
-      plt.plot(np.arange(N)/N, psi_f[0], 'b')
-  
-  plt.xlim(0, 0.5)
-  
-  plt.xlabel(r'$\omega$', fontsize=18)
-  plt.ylabel(r'$\hat\psi_j(\omega)$', fontsize=18)
-  plt.title('First-order filters (Q = {})'.format(Q), fontsize=18)
-  
-  ###############################################################################
-  # Do the same plot for the second-order filters. Note that since here `Q = 1`,
-  # we obtain wavelets that have higher frequency bandwidth.
-  
-  plt.figure()
-  plt.plot(np.arange(N)/N, phi_f[0], 'r')
-  for psi_f in psi2_f:
-      plt.plot(np.arange(N)/N, psi_f[0], 'b')
-  plt.xlim(0, 0.5)
-  plt.ylim(0, 1.2)
-  plt.xlabel(r'$\omega$', fontsize=18)
-  plt.ylabel(r'$\hat\psi_j(\omega)$', fontsize=18)
-  plt.title('Second-order filters (Q = 1)', fontsize=18)
-  
-  ###############################################################################
-  # Display the plots!
-  plt.show()
-
-def scattering1D_filter_factory_T_plot(N, J, T, phi_f, psi1_f, msg):
-  plt.figure()
-  plt.plot(np.arange(N)/N, phi_f[0], 'r', label=msg)
-  for psi_f in psi1_f:
-      plt.plot(np.arange(N)/N, psi_f[0], 'b')
-  plt.xlim(0, 0.5)
-  plt.xlabel(r'$\omega$', fontsize=18)
-  plt.ylabel(r'$\hat\psi_j(\omega)$', fontsize=18)
-  
-  plt.legend()
-  plt.show()
-
-def scattering1D_filter_factory_T():
-  ###############################################################################
-  #
-  # Constructs scattering filters, adjusting the T parameter which controls the
-  # amount of imposed time-shift invariance and maximum subsampling
-
-  N = 2**13
-  Q = 1       
-  sigma_low_scale_factor = [0, 5]        
-  Js = [5]
-  
-  for j in Js:
-      J = j
-      for i in sigma_low_scale_factor:
-          T=2**(J-i)
-          if i == 0:
-              default_str = ' (default)'
-          else: 
-              default_str = ''
-          phi_f, psi1_f, psi2_f, _ = scattering_filter_factory(np.log2(N), J, Q, T)
-          msg = 'J=' + str(J) + ', T=' +str(T) + default_str + ': LP-filter width $\sigma_{low}$=' + str(phi_f['sigma']) + ' (=0.1/T)'                
```

Comment:
Why do we not visualize this?

Reply:
Well, we could. For now I commited the other stuff.

Reply:
@janden @lostanlen are we fine with this?

Reply:
no. needs rebasing. `phi_f[0]` should be `phi_f['levels'][0]`, etc.

Reply:
this isn't #882-compliant